deploy/helm/charts/values.yaml

```diff
@@ -238,3 +238,11 @@ analytics:
   enabled: true
   # Specify in hours the duration after which a ping event needs to be sent.
   pingInterval: "24h"
+
+cleanup:
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: bitnami/kubectl
+    tag:
```

Comment:
Why the tag is empty? Are we going to use `latest` always?

Reply:
The tag is determined in the job template itself using the helm Capabilities to determine the k8s version, which will be same as the tag